lib/puppet-lint/plugins/top_scope_facts/top_scope_facts.rb

```diff
@@ -0,0 +1,35 @@
+# Public: A puppet-lint plugin that will check for the use of top scope facts.
+# For example, the fact `$facts['operatingsystem']` should be used over
```

Comment:
I would use a different fact as example, because that's a legacy fact and the legacy-fact-plugin will replace it with `$facts['os']['name']`

Reply:
@bastelfreak updated. Happy to use a completely different fact if you think there is a better example.